services/frontend-service/src/ui/components/navigation/navListItem.tsx

```diff
@@ -33,21 +44,24 @@ export const NavListItem = (props: { className?: string; to: string; icon?: JSX.
     }, []);
 
     return (
-        <Link
-            aria-label={to}
-            className={classNames(
-                'mdc-list-item',
-                pathname.startsWith(to) ? 'mdc-list-item--activated' : '',
-                className
-            )}
-            ref={control}
-            to={to}
-            tabIndex={pathname.startsWith(to) ? 0 : undefined}>
-            <div className="mdc-list-item__ripple" />
-            {icon &&
-                cloneElement(icon, {
-                    key: 'icon',
-                })}
-        </Link>
+        <div className="mdc-list-item-container" style={{ display: 'flex' }}>
```

Comment:
Please do not use inline styling move it to scss file
```suggestion
        <div className="mdc-list-item-container" >
```

Reply:
It is fixed